### DISCUSSION TITLE
Question: How i can make smooth change marker position

### DISCUSSION THREAD

**Discussion by alexdarh** [react-native-maps/react-native-maps#4448] [Q&A]
### Summary

Hi. i have my location marker on map. If i change my position on ios devices i have smooth  animation of changing position. On Android it not smooth, marker jump to new position. How i can get animation same as IoS

### Reproducible sample code

```TSX
question
```

### Steps to reproduce

question

### Expected result

question

### Actual result

question

### React Native Maps Version

1.3.0

### What platforms are you seeing the problem on?

Android

### React Native Version

0.69.1

### What version of Expo are you using?

Not using Expo

### Device(s)

Any android devices

### Additional information

Question

**Comment by SoloReverse** [comment]
Check the docs. theres a onMarkerPress method of the mapview that supplies the current coordinates of the marker. 
use that with animateToRegion().

`onMarkerPress={(e) =>
				map.current.animateToRegion(
					{
						...e.nativeEvent.coordinate,
						latitudeDelta: 0.1,
						longitudeDelta: 0.1,
					},
					1000
				)
			}`

**Comment by alexdarh** [comment]
> Check the docs. theres a onMarkerPress method of the mapview that supplies the current coordinates of the marker. use that with animateToRegion().
> 
> `onMarkerPress={(e) => map.current.animateToRegion( { ...e.nativeEvent.coordinate, latitudeDelta: 0.1, longitudeDelta: 0.1, }, 1000 ) }`

animateToRegion deprecated? Im right?
